### PR TITLE
Add Set-alapplayerrev for modifying version name and description of existing applayer versions

### PR DESCRIPTION
**Pull request by josephhu-personal**

### CODE REVIEW

**Comment by josephhu-personal** [comment]
It looks like AppVeyor thinks Set-ALAppLayerRev.ps1 is encoded in something other than UTF-8, but as far as I can tell by looking at the bottom bar in VSCode, it is. It says UTF-8 with BOM.
Would appreciate any guidance here. Thanks!

**Comment by ryancbutler** [comment]
Looking pretty good! And thank you!  We can bypass the UTF test.  Can you add your example to the README.md?

**Comment by josephhu-personal** [comment]
Updated README.md.

**Comment by ryancbutler** [comment]
Made a few minor changes to the readme and pretty printed the code. Also converted the ps1 to UTF-8 from UTF-8 BOM

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Alternative documentation can be found at https://unidesksdk.readthedocs.io/en/l
     + [New Application Layer Version](#new-application-layer-version)
     + [New Application Prerequisite Layer Version](#new-application-prerequisite-layer-version)
     + [Set Application Layer](#set-application-layer)
+    + [Set Application Layer Revision](#set-application-layer-revision)
```
Comment (josephhu-personal) [inline README.md:33]: whoops. forgot about this one. thanks!

**Pull request merged**